Doc: Convinient way to install clusteradm (#170)
open-cluster-management-io/open-cluster-management-io.github.io: content/en/getting-started/quick-start/_index.md

### Install clusteradm CLI tool

> It's recommended to install with the following command if you have a golang 

Reviewer: ```suggestion
It's recommended to install with the following command if you have a `golang` 
```

Author: quoted

environment:

```shell
$ go install open-cluster-management.io/clusteradm
```

> Otherwise if you don't have golang library in the local environment, you can also 

Reviewer: ```suggestion
Otherwise if you don't have a `golang` environment, you can also 
```
We should either use `golang` environment or `golang` library in the local environment but not both.

Author: fixed
